package.json

```diff
@@ -102,6 +102,7 @@
     "testPathIgnorePatterns": [
       "<rootDir>/frontend/"
     ],
+    "testTimeout": 30000,
```

Comment:
I had to set this timeout to get tests to pass in circle ci. We need to figure out why tests run so slow in CI.